### PR TITLE
#72 - Pull request

### PR DESCRIPTION
**Pull request by mcjazzyfunky**
@thysultan 
Please review the changes from branch "issue72".
Please let me know if there's something to change or fix.

Remarks:

- Some files use tabs instead of spaces (script/build/index.js etc) => I have not change that because it could (depending of the diff tool) cause some diff noise. Maybe you can convert the tabs whenever you like to do in future.

- In packages.json the "files" configuration lists all files of the "dist" directory explicitly. I've listed them explicitly as this was also done before. File patterns would also have been worked.

### CODE REVIEW

**Comment by thysultan** [comment]
Thanks. I noticed that [coveralls](https://coveralls.io/builds/18270565) reports on just the production builds, is this the same on your local machine?

File: package.json [hunk 1]
```diff
@@ -43,10 +43,10 @@
   "scripts": {
     "build": "node --harmony script/build",
     "build-watch": "node --harmony script/build -- --watch",
-    "test": "cross-env NODE_ENV=development nyc mocha --harmony --recursive --require script/test && cross-env NODE_ENV=production nyc mocha --harmony --recursive --require script/test",
+    "test": "cross-env NODE_ENV=development nyc mocha --harmony --recursive --require script/test && cross-env NODE_ENV=production nyc --no-clean mocha --harmony --recursive --require script/test",
     "test-watch-dev": "cross-env NODE_ENV=development mocha --harmony --recursive --require script/test --watch",
     "test-watch-prod": "cross-env NODE_ENV=production mocha --harmony --recursive --require script/test --watch",
-    "test-debug-dev": "cross-env NODE_ENV=development mocha --harmony --recursive --require script/test",
+    "test-debug-dev": "mocha --harmony --recursive --require script/test",
```
Comment (thysultan) [inline package.json]: Will this still run the development build?

**Comment by mcjazzyfunky** [comment]
@thysultan 
Thanks for that bug report regarding coveralls:
"npm test" runs nyc two times ... one for development, one for production ... had to use option "--no-clean" in the second run to prevent nyc from clearing the results from the first run.
=> is fixed ... could you please check again?

**Comment by thysultan** [comment]
@mcjazzyfunky Yes it now [reports](https://coveralls.io/builds/18275091) on both dev and prod builds.

File: package.json (hunk 1 above)
Comment (mcjazzyfunky) [reply package.json]: No that was a bug .. thanks for the info ... is fixed now

File: index.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+'use strict'
+
+if (process.env.NODE_ENV === 'development') {
+  module.exports = require('./dist/dio.cjs.development.min.js')
```
Comment (thysultan) [inline index.js]: I missed this before but the file name seems wrong, should be `dio.umd.development.min.js`. The `dio.cjs.development.min.js` entry is not a main entry point but the node server entry point. I guess the tests for `propTypes` will also serve to catch this.

Comment (mcjazzyfunky) [reply index.js]: I've changed this accordingly in branch "issue76":
https://github.com/thysultan/dio.js/commit/99dad6ec07b353521e36cc26bd19d11b1ccdada1#diff-168726dbe96b3ce427e7fedce31bb0bc

**Pull request merged**